SPECS/buildah/buildah.spec

```diff
@@ -140,14 +86,10 @@ cp imgtype %{buildroot}/%{_bindir}/%{name}-imgtype
 %dir %{_datadir}/bash-completion/completions
 %{_datadir}/bash-completion/completions/%{name}
 
-%files tests
-%license LICENSE
-%{_bindir}/%{name}-imgtype
-%{_datadir}/%{name}/test
-
 %changelog
-* Tue Nov 01 2022 Olivia Crain <oliviacrain@microsoft.com> - 1.18.0-6
-- Bump release to rebuild with go 1.18.8
+* Tue Nov 01 2022 Ameya Usgaonkar <ausgaonkar@microsoft.com> - 1.18.0-6
+- Move to core packages
+- Remove tests package
```

Comment:
Please leave the old changelog entry and increment the `Release` tag instead.

Reply:
Strange! I am suspecting role of spec-cleaner here.  Thanks for pointing out.